libs/maps/src/maps/CPointsMap_liblas.cpp

```diff
@@ -113,8 +113,8 @@ bool CPointsMap::loadLASFile(const std::string &filename, LAS_HeaderInfo &out_he
 
 	out_headerInfo.FileSignature      = header.GetFileSignature();
 	out_headerInfo.SystemIdentifier   = header.GetSystemId();
-	out_headerInfo.SoftwareIdentifier = header.GetSoftwareId();
-	out_headerInfo.project_guid       = header.GetProjectId().to_string();
+	out_headerInfo.SoftwareIdentifier = header.
+	out_headerInfo.project_guid       = boost::lexical_cast<std::string>(header.GetProjectId());
```

Comment:
no problem! :+1: 
If possible, please check that everything works after this PR in `master` and let us know if you find further issues...

Cheers!